docs/src/components/home/GoldSponsors.tsx

```diff
@@ -11,56 +11,56 @@ const GOLDs = [
     srcSet: 'https://avatars.githubusercontent.com/u/30204434?s=80 2x',
     name: 'Tidelift',
     description: 'Enterprise-ready open source software.',
-    href: 'https://tidelift.com/',
+    href: 'https://tidelift.com/?utm_medium=referral&utm_source=MUI&utm_content=homepage',
```

Comment:
Same feedback as before, we need the special UTMs Tidelift gave us. The source of truth is the URL we use for the Tidelift ad.